google_workgroup/main.tf

```diff
@@ -0,0 +1,93 @@
+/**
+ * # workgroup
+ * Retrieve workgroup ACL lists associated with data and gcp access workgroups.
+ *
+ * Workgroup identifiers should be of the form:
+ *
+ * ```
+ * workgroup:WORKGROUP_NAME[/SUBGROUP]
+ * ```
+ *
+ * where `SUBGROUP` defaults to `default`. For example: `workgroup:app`, `workgroup:app/admin`.
+ *
+ * For subgroup queries across all workgroups, an additional identifier format:
+ *
+ * ```
+ * subgroup:SUBGROUP
+ * ```
+ *
+ * is supported, which will return all workgroups that contain a particular subgroup.
+ *
+ * This module is cloned from https://github.com/mozilla-services/cloudops-infra-terraform-modules/tree/master/data-workgroup.
+ *
+ */
+
+data "terraform_remote_state" "workgroups" {
```

Comment:
This is simpler than https://github.com/mozilla-services/cloudops-infra-terraform-modules/commit/ff6c3035468fc003d58ecee5c14247401240985b, I guess I did it this way before because the primary use case was local testing (no remote state)